torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -2441,29 +2441,6 @@ class IrParser {
           nullptr);
     }
 
-    {
```

Comment:
Looks like we don't need to add profiling for linear now. Wondering if some other pass were doing that. Removing this as a whole *should* break `TestCudaFuser.test_linear` but it apparently didn't :)

I'm thinking more like this one to be on the safer side. (waiting on build to test this)
```
https://github.com/pytorch/pytorch/pull/75897/files#diff-9738ee51d55cdf479f41b018d61ca43ed2ae10c13a20e21ec45b7578e82a23d2R2457-R2460
```

Reply:
double checked this on my local build. ProfileRegistry is profiling all differentiable node so we don't have to register linear for profiling in nvfuser. But I still do prefer to have it registered just to be on the safe side.

Nevertheless, since this is not blocking, I'm stamping it